src/components/Sidebar/Sidebar.jsx

```diff
@@ -34,8 +34,12 @@ class Sidebar extends React.Component {
     this.handleLogoutClick = this.handleLogoutClick.bind(this);
   }
 
+  componentDidMount(){
+    this.state.activeState.dashboard === true && this.props.history.push(`/dashboard`);
+  }
+
   handleClick(link) {
-    if (this.state.activeState[link]) {
+    if (this.state.activeState[link] === false) {
```

Comment:
similarly, is there a reason for === false? Is this object going to be anything else but boolean

Reply:
because this.state.activeState[link] alone just means if it exists. so it doesn't enter the if statement.

Reply:
I added === false so that if I click on another nav it will redirect

Reply:
`=== false` is definitely redundant here.

Reply:
Not true, its neccesary. if its not there it wont enter the if block. and if you do !this.state.activeState[link] it doesnt mean its false, it means is not a key of activeState

Reply:
I find that a bit odd 

if it's a boolean value 

`this.state.activeState[link]: false` 

`(!this.state.activeState[link])` would go in, if it's true it wouldn't. The only time that you should need to explicitly write === false is because you're saying that this.state.activeState[link] can be a value other boolean (such as undefined, null, object, etc), is that the case?

Reply:
This also passes the test if you try to pass in a random string as link in the handle function. Since that random string is not a key in the active state it will not enter the if block.

Reply:
okay which means there's the possibility in which the value can not be a boolean? Only then is it fine to do this equals (but even then to me it's odd that the value could not be boolean value, but i digress)

Reply:
IMO i think the activeState should be an stored as a string value (kind of like how we do for activeTab). That way, we can more accurately pinpoint the activeState.

Doing it this way where a `boolean` or a random string are potential params is not so intuitive.